climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -83,6 +86,18 @@
 MISSING_LONG = ec.CODES_MISSING_LONG
 """Missing double and integers in ecCodes """
 
+CXML2CSV_XSL = Path(__file__).parent / "tc_tracks_foreast_cxml2csv.xsl"
+"""Path at which an xsl is found for transforming CXML to CSV format."""
```

Comment:
do we want this file here or in the climada/data folder? @emanuel-schmid

Reply:
Hi @emanuel-schmid, here is one issue.

Reply:
We certainly don't want it here. Data folder maybe. Or climada data api. Depends on what is that file. What's inside, who wrote it and how, will it ever change, can we have it as csv file instead?

Reply:
@emanuel-schmid No, it's not a data file; it's XSL, so it contains transformation logic.